File: app/src/main/java/com/ltst/instagramgallerysample/utils/Logger.java
package com.ltst.instagramgallerysample.utils;

import android.util.Log;

public class Logger {

    private static final String TAG = "InstagramGallarySample";
    private static boolean isDebugable = false;

    public static void d(String message) {
        Log.d(TAG, message);
    }

    public static void d(String message, Object... args) {
        Log.d(TAG, String.format(message, args));
    }

    public static void setDebugable(boolean debugable) {
        isDebugable = debugable;
    }

}

File: library/src/main/java/com/cosic/instagallery/utils/Logger.kt
package com.cosic.instagallery.utils

import android.util.Log

object Logger {

    private val TAG = "InstagramGallarySample"
    private var isDebugable = false

    fun d(message: String) {
        Log.d(TAG, message)
    }

    fun d(message: String, vararg args: Any) {
        Log.d(TAG, String.format(message, *args))
    }

    fun setDebugable(debugable: Boolean) {
        isDebugable = debugable
    }

}
